components/processing/form_parser/src/main.py

```diff
@@ -52,10 +53,12 @@ def batch_process_documents(
         gcs_output_uri: GCS directory to store the out json files,
         gcs_input_prefix: GCS directory to store input files to be processed
     """
+  USER_AGENT = "cloud-solutions/eks-agent-builder-v1"
```

Comment:
Please use the following value for the USER_AGENT string:
"cloud-solutions/eks-docai"

Reply:
updated